pkg/cephfs/volume.go

```diff
@@ -31,11 +31,11 @@ const (
 )
 
 var (
-	cephRootPrefix = PluginFolder + "/controller/volumes/root-"
+	cephRootPrefix = "/controller/volumes/root-"
 )
 
 func getCephRootPathLocal(volID volumeID) string {
-	return cephRootPrefix + string(volID)
+	return PluginFolder + cephRootPrefix + string(volID)
```

Comment:
as `cephRootPrefix` is only referred inside this function, can we just remove `cephRootPrefix` from global declaration and return `fmt.Sprintf("%s/controller/volumes/root-%s,PluginFolder,string(volID))`

Reply:
define as  a const string may be  a better way, it can explict describe how CephRootPath contstruct